app/bankaccountverification/testOnly/TestSetupController.scala

```diff
@@ -37,9 +39,44 @@ class TestSetupController@Inject()(
   
   private lazy val unAuthRedirect = Future.successful(Redirect(appConfig.authStubUrl + s"/?continue=${appConfig.testOnlyUrl}${bankaccountverification.testOnly.routes.TestSetupController.show().url}")) 
   
+  val fullExampleJson = Json.prettyPrint(Json.toJson(InitRequest(
+    serviceIdentifier = "bank-account-verification-frontend",
+    continueUrl = "bank-account-verification/test-only/complete",
+    prepopulatedData = Some(InitRequestPrepopulatedData(
+      accountType = Business,
+      name = Some("British Airways Ltd"),
+      sortCode = Some("010013"),
+      accountNumber = Some("03766284"),
+      rollNumber = Some("1234567890")
+    )),
+    address = Some(InitRequestAddress(
+      lines = List("Line1", "Line2"),
+      town = Some("Town"),
+      postcode = Some("AA00AA")
+    )),
+    messages = Some(InitRequestMessages(
+      en = Json.obj("key1" -> "value1", "key2" -> "value2"),
+      cy = Some(Json.obj("key1" -> "gwerth1", "key2" -> "gwerth2"))
```

Comment:
Can you swap this for this:
```
Json.obj(
      "service.name" -> "My service"
      "label.accountDetails.heading.business" -> "Business bank or building society account details",
      "label.accountDetails.heading.personal" -> "Personal bank or building society account details",
    ))
```

It's taken from the readme but uses actual message keys. At least then it's a more realistic example :)